cockroachcloud/sessions-page.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Sessions Page
+summary: The Sessions page provides details of all open sessions in the cluster.
+toc: true
+---
+
+The **Sessions** page of the CockroachCloud Console provides details of all open sessions in the cluster.
+
+To view this page, click **Sessions** in the left-hand navigation of the CockroachCloud Console.
+
+## Sessions list
+
+Use the **Sessions** list to see the open sessions in the cluster. This includes active and idle sessions.
+
+{{site.data.alerts.callout_info}}
+A session is *active* if it has an open transaction (including implicit transactions, which are individual SQL statements), and *idle* if it has no open transaction. Active sessions consume hardware resources.
+{{site.data.alerts.end}}
```

Comment:
This looks good. I might suggest one small improvement. I'm not seeing **active** and **idle** in the UI screenshot. Am I correct in assuming:

1. the terms are not actually displayed anywhere
2. that N/A is actually shown in the age columns for *idle* sessions?

If that's all true, then add point number 2 for clarity? (Or any other ways to identify which state that I'm missing.)